bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use CubeTools\CubeCommonDevelop\SymfonyCommands;
+use CubeTools\CubeCommonDevelop\Workplace\DevCommands;
```

Comment:
exactly seen this is on the next command

Reply:
no , it is on line 13 (https://github.com/EmchBerger/cube-common-develop/pull/44/commits/a419ce9783fcd98de77ca7a80c88bc3e6320701b#diff-834e3741acba32abc672bec6be613b2fR13)